### PR TITLE
Add MX, TXT Records + Add Domains - Dashboard

### PR DESCRIPTION
**Pull request by ukriu**
## Overview

Add sub-domains `email`, `secret`,  Reason given for `secret`.
Add MX and TXT Records to some existing domains.
---
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/email.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ukrioo",
+        "email": "u8k50850@gmail.com"
+    },
+    "record": {
+        "URL": "https://ukrioo.github.io/"
```
Comment (SX-9) [inline domains/email.json:7]: note: we are having issues with url record so if its not working as expected contact on discord

**Review by satr14washere — APPROVED** [review]

File: domains/email.json
Comment (DEV-DIBSTER) [inline domains/email.json:1]: Will not release this domain as it has the potential for abuse.

File: domains/secret.json
Comment (DEV-DIBSTER) [inline domains/secret.json:1]: Will not release this domain as it has the potential for abuse.

File: domains/uk.json
Comment (DEV-DIBSTER) [inline domains/uk.json:1]: Will not release this domain as it has the potential for abuse.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**